### ISSUE TITLE
Issue when upgrading to androidX

### ISSUE THREAD

**Issue by lucidtheory** [Instabug/Instabug-React-Native#231]
on instabug version 8.0.19
react native 57.8

My project is upgraded to androidX. I noticed this issue in the android repository:
https://github.com/Instabug/Instabug-Android/issues/123

And was curious if the two issues were related.

Basically, when trying to build a release build, I get this issue:

```
> Task :instabug-reactnative:compileReleaseJavaWithJavac FAILED
/client/node_modules/instabug-reactnative/android/src/main/java/com/instabug/reactlibrary/RNInstabugReactnativeModule.java:7: error: package android.support.annotation does not exist
import android.support.annotation.Nullable;
                                 ^
/client/node_modules/instabug-reactnative/android/src/main/java/com/instabug/reactlibrary/RNInstabugReactnativeModule.java:2161: error: cannot find symbol
                           @Nullable WritableMap params) {
                            ^
  symbol:   class Nullable
  location: class RNInstabugReactnativeModule
Note: Some input files use or override a deprecated API.
Note: Recompile with -Xlint:deprecation for details.
Note: Some input files use unchecked or unsafe operations.
Note: Recompile with -Xlint:unchecked for details.
2 errors


FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':instabug-reactnative:compileReleaseJavaWithJavac'.
> Compilation failed; see the compiler error output for details.

```

Instabug is looking for android.support.annotation.nullable and failing to resolve. 

In gradle.properties I have 
```
android.useAndroidX=true
android.enableJetifier=true
```
as per the androidX upgrade requirements. Not sure how to solve.

**Comment by salmamali** [comment]
@lucidtheory We are working on the issue and will let you know when we release the fix.

**Comment by HeshamMegid** [comment]
@lucidtheory closing this issue since we have a conversation open with you over email.

**Comment by soumyamishra89** [comment]
@HeshamMegid i am facing the exact same issue when enabling androidx. Can you help me with the issue?

**Comment by lucidtheory** [comment]
I am still getting the same error after upgrading instabug to `8.0.20`

**Comment by salmamali** [comment]
@lucidtheory This issue is now fixed in v8.0.21. Please update to the latest.
